### DISCUSSION TITLE
Missing support for .... ah if reading though all discussions it's already there .. found by chance

### DISCUSSION THREAD

**Discussion by commentsonly** [mak-gitdev/HA_enoceanmqtt#14] [Ideas]
Could you please think about an easy way to inform about updated mapping.yaml files?
After my ordered FTKWs arrived today I though about editing the mapping.yaml I fetched last week but since i was lazy (what a luck) I first though about browsing the discussions and scoobidoo ... F6-10-00 already implemented. Saved me some minutes.

I'm not sure if the HA addon construct would allow for a second hyperlink next to the "(Changelog)" one so that you could fire up a window which simply compared the local mapping.yaml against the latest version on github.
I wouldn't ask for signalling a version update since then you might have to signal one 2-3 times each day.

**Comment by mak-gitdev** [comment]
Hi,

Good point there.

I may open a discussion that will list the available EEPs not yet integrated in the master branch and the one on the To Do list as well.
You will then just have to have a look at this list.

I am also thinking about starting a dev branch of the addon that will integrate more quickly new EEPs as well as bugfix and feature requests.
